src/deals/list/deals.ts

```diff
@@ -152,6 +152,15 @@ export class Deals {
     return !!this.getDealsForCardIndex(cardIndex, showMine, address)?.length;
   }
 
+  /**
+   * Gets the title display text for the all deals grid
+   */
+  @computedFrom("showMine", "cardIndex")
+  get dealDisplayText() : string{
```

Comment:
@dkent600 I had it in the view model originally, but it was a quadruple ternary so I thought it would simplify it to put it into a getter. I can put it back to the quadruple ternary if you would like though.

Reply:
@dkent600 Nevermind, I realized that I could do it in two ternary expressions and added it to the view.